src/class/object_genome_variants.php

```diff
@@ -173,6 +173,12 @@ function __construct ()
                         'chromosome' => array(
                                     'view' => array('Chr', 50),
                                     'db'   => array('vog.chromosome', 'ASC', true)),
+                        'position_g_start' => array(
+                                    'view' => false,
+                                    'db'   => array('position_g_start', 'ASC', 'INT')),
+                        'position_g_end' => array(
+                                     'view' => false,
+                                    'db'   => array('position_g_end', 'ASC', 'INT')),
```

Comment:
Adding `vog.` to the database field name and removing `'INT'` breaks things. Somehow the position search conditions are ignored.

I've left this untouched for now.